{"message":"error happened","stack":"Error: error happened\n    at E:\\Temp\\node\\vidly\\routers\\users.js:13:11\n    at newFn (E:\\Temp\\node\\vidly\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (E:\\Temp\\node\\vidly\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (E:\\Temp\\node\\vidly\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (E:\\Temp\\node\\vidly\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at newFn (E:\\Temp\\node\\vidly\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (E:\\Temp\\node\\vidly\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at E:\\Temp\\node\\vidly\\node_modules\\express\\lib\\router\\index.js:281:22\n    at Function.process_params (E:\\Temp\\node\\vidly\\node_modules\\express\\lib\\router\\index.js:341:12)\n    at next (E:\\Temp\\node\\vidly\\node_modules\\express\\lib\\router\\index.js:275:10)","level":"error","timestamp":"2022-01-19T14:28:06.925Z"}
